controllers/advertismentController.js

```diff
@@ -0,0 +1,13 @@
+const Advertisement = require('../models/advertisment');
+
+exports.submitAdvertisement = async (req, res) => {
+  const { company_name, title, duration, job_type, location, email, description } = req.body;
+  const newAdvertisement = new Advertisement({ company_name, title, duration, job_type, location, email, description });
+
+  try {
+    await newAdvertisement.save();
+    res.status(200).send('Post submitted successfully!');
```

Comment:
Suggestion: I think status code 201 is more suitable here.

Reply:
Thank you for the suggestion! I'll change the status code to 201, as it better indicates successful resource creation.